src/main/java/controllers/TicketManagementController.java

```diff
@@ -14,21 +16,51 @@
 import javax.servlet.http.HttpServletResponse;
 import java.io.IOException;
 import java.sql.SQLException;
-import java.util.ArrayList;
-import java.util.Set;
+import java.util.*;
 
 @WebServlet("/ticketmanagement")
 public class TicketManagementController extends HttpServlet {
 
     private Set<Flight> flights;
-    private FlightDAO flightDAO = new FlightDAO();
-    private TicketDAO ticketDAO = new TicketDAO();
-    static final Logger LOGGER = Logger.getLogger(TicketManagementController.class);
+    private final FlightDAO flightDAO = new FlightDAO();
+    private final TicketDAO ticketDAO = new TicketDAO();
+    private final PlaneDAO planeDAO = new PlaneDAO();
+
+    private final Set<Ticket> setOfTickets = new HashSet<>();
+    private final Set<Plane> setOfPlanes = new HashSet<>();
```

Comment:
your fild names should tell what is this fild for. I can see that this is set of planes without this name